doc/sphinx/proof-engine/tactics.rst

```diff
@@ -264,6 +264,11 @@ Applying theorems
       This tactic behaves like :tacn:`simple refine` except it performs type checking
       without resolution of typeclasses.
 
+   .. flag:: Debug Unification
+
+   Enables printing traces of unification steps used during
```

Comment:
Indentation problem.

Reply:
I indented it because it appears to apply only to `refine', which is the tactic before this line (about 2 pages).  There are 5 existing flags that have similar indentation, all in this file.  You want to change all of them to be at the left margin?

Reply:
I am not talking about the indentation of `.. flag:: Debug Unification` but the indentation of its documentation:
```suggestion
      Enables printing traces of unification steps used during
```